catroid/src/androidTest/java/org/catrobat/catroid/test/common/LookDataTest.java

```diff
@@ -23,23 +23,84 @@
 
 package org.catrobat.catroid.test.common;
 
+import android.support.test.InstrumentationRegistry;
 import android.support.test.runner.AndroidJUnit4;
+import android.util.Log;
 
 import com.badlogic.gdx.graphics.Pixmap;
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.TextureRegion;
 
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.common.Constants;
 import org.catrobat.catroid.common.LookData;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.SingleSprite;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.io.ResourceImporter;
+import org.catrobat.catroid.io.StorageOperations;
+import org.catrobat.catroid.io.XstreamSerializer;
+import org.catrobat.catroid.utils.ImageEditing;
+import org.catrobat.catroid.utils.PathBuilder;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import java.io.File;
+import java.io.IOException;
+
+import static junit.framework.Assert.assertEquals;
+
+import static org.catrobat.catroid.common.Constants.IMAGE_DIRECTORY_NAME;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-
 @RunWith(AndroidJUnit4.class)
 public class LookDataTest {
 
+	@Test
+	public void testCollisionInformation() throws IOException, InterruptedException {
+		final String expectedMetadata = "1.0;94.0;5.0;143.0;25.0;119.0;35.0;150.0;73.0;138.0;92.0;150.0;100.0;118.0;"
+				+ "110.0;143.0;124.0;141.0;120.0;88.0;106.0;105.0;21.0;101.0;105.0;98.0;105.0;39.0;82.0;0.0;68.0;"
+				+ "29.0;52.0;0.0;38.0;17.0;7.0;19.0;27.0;31.0;22.0;95.0";
+
+		Project project = new Project(InstrumentationRegistry.getTargetContext(), "testProject");
+
+		final String filePath = project.getDefaultScene().getDirectory() + "/" + IMAGE_DIRECTORY_NAME + "/collision_donut.png";
+
+		Sprite sprite = new SingleSprite("testSprite");
+		project.getDefaultScene().addSprite(sprite);
+
+		ProjectManager.getInstance().setProject(project);
+		ProjectManager.getInstance().setCurrentSprite(sprite);
+		XstreamSerializer.getInstance().saveProject(project);
+
+		File imageFile = ResourceImporter.createImageFileFromResourcesInDirectory(
+				InstrumentationRegistry.getContext().getResources(),
+				org.catrobat.catroid.test.R.drawable.catroid_banzai,
```

Comment:
this is not loading the collision_donut resource, its loading the catroid_banzai image and renaming the destination file to collision_donut